core/src/main/scala/org/apache/spark/api/r/RBackendHandler.scala

```diff
@@ -69,8 +69,11 @@ private[r] class RBackendHandler(server: RBackend)
             case e: Exception =>
               logError(s"Removing $objId failed", e)
               writeInt(dos, -1)
+              writeString(dos, s"Removing $objId failed: ${e.getMessage}")
           }
-        case _ => dos.writeInt(-1)
+        case _ =>
+          dos.writeInt(-1)
+          writeString(dos, "Unknown error")
```

Comment:
Isn't this an unknown method call? Should this say "error unknown method" or similar?